### PR TITLE
validate signature size on SendTransactionWithSignature

### PR DESCRIPTION
**Pull request by gravityblast**
Check signature size to avoid panic from go-ethereum

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]
A test would be nice, but looks good otherwise.

**Comment by gravityblast** [comment]
thanks @mandrigin , I added a test, waiting now for CI

File: transactions/transactor.go [hunk 1]
```diff
@@ -24,8 +25,13 @@ const (
 	sendTxTimeout = 300 * time.Second
 
 	defaultGas = 90000
+
+	validSignatureSize = 65
```
Comment (adambabik) [inline transactions/transactor.go:29]: Can we take this value from go-ethereum package?

**Review by adambabik — APPROVED** [review]

File: transactions/transactor.go (hunk 1 above)
Comment (gravityblast) [reply transactions/transactor.go:29]: @adambabik I see it's actually hardcoded there: 

https://github.com/status-im/status-go/blob/develop/vendor/github.com/ethereum/go-ethereum/core/types/transaction_signing.go#L196

**Pull request merged**